frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Tab-Sidebar-ODC-4116.feature

```diff
@@ -16,6 +18,22 @@ Scenario: Monitoring tab on the Sidebar for Deployments: MH-03-TC02
     Then user will be taken to Dashboard tab on the Monitoring page
 
 
+@regression
+Scenario: Monitoring tab on the Sidebar for Helm Release: MH-03-TC02
+    Given user is at Topology page
+    When user clicks on the workload "node-js-ex" to open the sidebar
+    And user clicks on Monitoring tab
+    And user clicks on View Monitoring dashborad link
+    Then user will be taken to Dashboard tab on the Monitoring page
+
+
+@regression
+Scenario: Monitoring tab on the Sidebar for Knative Service: MH-03-TC02
+    Given user is at Topology page
+    When user clicks on the knative service "knative-demo" to open the sidebar
+    Then user wont see Monitoring tab
```

Comment:
*won't

Reply:
@makambalaji asked me to use wont instead of won't.

Reply:
Ah.. didn't saw that comment